sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactController.java

```diff
@@ -536,6 +551,40 @@ public CommitDiscardWrapperComponent<ContactCreateForm> getContactCreateComponen
 		return createComponent;
 	}
 
+	public CommitDiscardWrapperComponent<ContactCreateForm> getContactCreateComponent(PersonDto person) {
+		final ContactCreateForm createForm;
+
+		createForm = new ContactCreateForm(null, false, false, true);
+		createForm.setValue(createNewContact(person));
+		createForm.setPerson(person);
+		createForm.setPersonDetailsReadOnly();
+		createForm.setDiseaseReadOnly();
```

Comment:
why when creating a contact from person the disease cannot be set? The only way to create a contact is to select a source case.

Reply:
Done.